CONTRIBUTING.md

```diff
@@ -144,3 +169,44 @@ config.ichain_force_test_username = "testuser"
 # set email of 'testuser'
 config.ichain_force_test_attributes = {:email => "testuser@example.com"}
 ```
+## Labels for issues and PRs
```

Comment:
Great! This is really useful! :tada: 

Although, what are there some tags without description?